### PR TITLE
Issue #14436: Enforced naming convention on MagicNumberCheckTest

### PR DESCRIPTION
**Pull request by Pinti0001**
Issue #14436 

Enforced naming convention on MagicNumberCheckTest

### CODE REVIEW

**Review by MANISH-K-07 — COMMENTED** [review]
@Pinti0001 ,
Please do update all the input files and related test methods of this module.

> Each PR must contain one complete module with all existing filenames and test method names analysed and renamed.
> In update, naming should be such that, just by looking at the names, anyone can figure out what the test validates from input file without even having to open it.

You can go through the issue #14436 description again.
It is recommended to pick one module and work on it completely. This helps to avoid confusion for future contributors who happen to pick  the same module if it is merged half done.

**Comment by Pinti0001** [comment]
> @Pinti0001 , Please do update all the input files and related test methods of this module.
> 
> > Each PR must contain one complete module with all existing filenames and test method names analysed and renamed.
> > In update, naming should be such that, just by looking at the names, anyone can figure out what the test validates from input file without even having to open it.
> 
> You can go through the issue #14436 description again. It is recommended to pick one module and work on it completely. This helps to avoid confusion for future contributors who happen to pick the same module if it is merged half done.

ok @MANISH-K-07  got it I am trying to complete module

**Comment by Pinti0001** [comment]
Hey @romani  @MANISH-K-07  can you check that is name correct or not....

**Comment by MANISH-K-07** [comment]
Why haven't the Checkstyle pipelines workflow run here ? 🙄

**Comment by Pinti0001** [comment]
I was also surprised to see this,
I never saw anything like this before.

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberCheckIgnoreFieldDeclarationRecords.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 //non-compiled with javac: Compilable with Java14
 package com.puppycrawl.tools.checkstyle.checks.coding.magicnumber;
 
-public class InputMagicNumberIgnoreFieldDeclarationRecords {
+public class InputMagicNumberCheckIgnoreFieldDeclarationRecords {
```
Comment (romani) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberCheckIgnoreFieldDeclarationRecords.java:18]: `Check` should be required, it is implied. let keep name shorted

**Review by romani — CHANGES_REQUESTED** [review]
items:

**Comment by romani** [comment]
I am really sorry, we are cancelling this issue.

**Pull request closed without merging**